README.md

```diff
@@ -1,2 +1,33 @@
 # pfb
-osm2pgrouting mapconfig file
+
+## Docker
+
+To run the analysis in docker, first build the docker image:
+
+```bash
+docker build -t pfb .
+```
+
+Then run the analysis as follows:
+
+```bash
+docker run \
+    -e PFB_SHPFILE=/data/neighborhood_boundary.shp \
+    -e PFB_STATE=ma \
+    -e PFB_STATE_FIPS=25 \
+    -e NB_INPUT_SRID=2249 \
+    -e NB_OUTPUT_SRID=2249 \
+    -e NB_BOUNDARY_BUFFER=11000 \
+    -v /home/user/projects/pfb-network-connectivity/data/:/data/ \
```

Comment:
Addressed in https://github.com/azavea/pfb/pull/15/commits/b6115bb9d6055c2ca853f915f658e36caac70dcb